src/goto-programs/goto_program.h

```diff
@@ -441,7 +441,10 @@ class goto_programt
         source_location(std::move(_source_location)),
         type(_type),
         guard(std::move(_guard)),
-        targets(std::move(_targets))
+        targets(std::move(_targets)),
+        location_number(0),
+        loop_number(0),
+        target_number(nil_target)
```

Comment:
Let's instead place those initialisers at the declaration of each of the members so that we don't have to do this in each an every constructor, which is just bound to us making more of the same mistakes.

Reply:
ok